src/lib.rs

```diff
@@ -430,6 +418,15 @@ impl LightningNode {
     }
 }
 
+fn to_breez_env(network: Network) -> EnvironmentType {
+    match network {
+        Network::Bitcoin => EnvironmentType::Production,
+        Network::Testnet => EnvironmentType::Staging,
+        Network::Signet => EnvironmentType::Staging,
+        Network::Regtest => EnvironmentType::Staging,
+    }
+}
```

Comment:
It isn't clear from the SDK docs, but I looked at the [source code](https://github.com/breez/breez-sdk/blob/b09301fd865ca4ee234c299df7df9539235afbbf/libs/sdk-core/src/models.rs#L486), and it seems both `Production` and `Staging` use `Network::Bitcoin`, so we can only support mainnet.

Reply:
Added the `environment_type` field to our Environment Configuration and switched the Network to Bitcoin for all.